### PR TITLE
feat: Add buttons to add all roles and clear all roles for user

### PR DESCRIPTION
**Pull request by meshramsaroj**
**Task Link:** https://app.asana.com/0/1194051119161697/1199997172288575/f

**Description:** Display add_all_roles and clear_all_roles buttons on user page

**Gif:**
add_or_clear_all_roles

### CODE REVIEW

File: frappe/public/js/frappe/roles_editor.js [hunk 1]
```diff
@@ -22,30 +22,29 @@ frappe.RoleEditor = Class.extend({
...
-
-			role_toolbar.find(".btn-add")
-				.html(__('Add all roles'))
-				.on("click", function() {
-					$(me.wrapper).find('input[type="checkbox"]').each(function(i, check) {
-						if (!$(check).is(":checked")) {
-							check.checked = true;
-						}
-					});
+		var role_toolbar = $('<p><button class="btn btn-default btn-add btn-sm" style="margin-right: 5px;"></button>\
```
Comment (hrwX) [inline frappe/public/js/frappe/roles_editor.js]:
```suggestion
		let role_toolbar = $('<p><button class="btn btn-default btn-add btn-sm" style="margin-right: 5px;"></button>\
```

**Review by hrwX — APPROVED** [review]

**Review by nehasacher143 — COMMENTED** [review]
I guess there is only an 'if statement' removed @hrwX @meshramsaroj

**Comment by hrwX** [comment]
> I guess there is only an 'if statement' removed @hrwX @meshramsaroj

Yes that is correct. And it works perfectly fine since they were hidden for user doctype

**Review by nehasacher143 — APPROVED** [review]
Looks okay :)

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**